src/Mod/PartDesign/App/FeatureAddSub.h

```diff
@@ -40,6 +40,11 @@ class PartDesignExport FeatureAddSub : public PartDesign::Feature
         Subtractive
     };
 
+    enum RefineErrorPolicy {

```

Comment:
I think you can make this an `enum class` with no further code changes, since you explicitly qualify your uses below.
```suggestion
    enum class RefineErrorPolicy {
```